### PR TITLE
feat(curriculum): add travel agency page lab

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+assert($('ul')[0]?.previousElementSibling.tagName === 'P');
+```
+
+You should have two items in your unordered list.
+
+```js
+assert.equal(document.querySelectorAll('li').length,  2);
+```
+
+Both your list items should contain an anchor element.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md:119]: This test is passing without code.

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+assert.equal($('h2')[1]?.innerText, 'Top Itineraries');
+```
+
+You should have at least three `figure` elements.
+
+```js
+assert.isAtLeast(document.querySelectorAll('figure').length, 3)
+```
+
+Each `figure` element should contain an anchor element as its first child.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md:157]: Tests from here onwards are always passing.

**Comment by Dario-DC** [comment]
Tests should work now @zairahira

**Review by zairahira — APPROVED** [review]
The tests are working as expected 👍️

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -1713,6 +1709,12 @@
       "front-end-development-certification-exam": {
         "title": "Front End Development Certification Exam",
         "intro": ["", ""]
+      },
+      "lab-travel-agency-page": {
+        "title": "Travel Agency Page",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:1714]:
```suggestion
        "title": "Build a Travel Agency Page",
```

File: curriculum/challenges/_meta/lab-travel-agency-page/meta.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Travel Agency Page",
```
Comment (moT01) [inline curriculum/challenges/_meta/lab-travel-agency-page/meta.json:2]:
```suggestion
  "name": "Build a Travel Agency Page",
```

**Review by moT01 — CHANGES_REQUESTED** [review]
This looks good @Dario-DC 👍 I left a few suggestions to adjust the block name. Other than that, I would be fine getting this in. I do think it could be a little better by having even more flexibility in what users can use for the content. Instead of forcing certain text, let them choose - but there's a numbers of things where they aren't forced to use specific text, so there's a decent amount of flexibility 👍

Edit: also, it looks there's a merge conflict right now.

**Comment by Dario-DC** [comment]
Title and merge conflicts are fixed. @moT01 If you think there's not enough flexibility I can change a couple of user stories.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**